### ISSUE TITLE
Submit on KITTI website

### ISSUE THREAD

**Issue by JLIN77** [open-mmlab/OpenPCDet#1116]
Yes. I did inference the KITTI test set with official PV-RCNN weight, and submit the results to KITTI website. The results show as:
Car (3D Detection) | 66.01 % | 64.26 % | 61.65 %
Pedestrian (3D Detection) | 35.58 % | 29.18 % | 27.36 %
Cyclist (3D Detection) | 51.01 % | 42.79 % | 39.31 %
Compared with mmlab-PVRCNN model on KITTI leaderboard, it drops almost 10~20 mAP.
Any answer to this question?

**Comment by sshaoshuai** [comment]
It seems strange, can you fisrt evaluate the performance on KITTI validation set? Generally the performance between test and validation will not be very large like that.

**Comment by MLbeginner2** [comment]
I met the same problem.I did inference the KITTI test set with official PointPillar_7728.pth, and submit the results to KITTI website. The results show as:
Car (3D Detection) | 82.31 % | 72.98 % | 66.73 %
Pedestrian (3D Detection) | 40.65 % | 32.74 % | 30.19 %
Cyclist (3D Detection) | 69.39 % | 54.12 % | 47.82 %
 This is too strange, could it be that the pcdet is not sufficiently trained?

**Comment by MLbeginner2** [comment]
> I met the same problem.I did inference the KITTI test set with official PointPillar_7728.pth, and submit the results to KITTI website. The results show as: Car (3D Detection) | 82.31 % | 72.98 % | 66.73 % Pedestrian (3D Detection) | 40.65 % | 32.74 % | 30.19 % Cyclist (3D Detection) | 69.39 % | 54.12 % | 47.82 % This is too strange, could it be that the pcdet is not sufficiently trained?
I am training on all KITTI data with labels, not 3712.

**Comment by BravoCharlie-ai** [comment]
Hello guys,

Can you let me know how you convert pickle file to submit on leaderboard?
Do I need to do the coordinate transformation? I uploaded the result but I got zero for 3D detection. Whereas Bbbox results are sufficiently high?

**Comment by MLbeginner2** [comment]
> Hello guys,
> 
> Can you let me know how you convert pickle file to submit on leaderboard? Do I need to do the coordinate transformation? I uploaded the result but I got zero for 3D detection. Whereas Bbbox results are sufficiently high?

run python test.py ********** --save_to_file

**Comment by BravoCharlie-ai** [comment]
Thank you for the reply. But I still face the same issue. I even don't see 3D detections in the leaderboad result.


![Screenshot from 2023-02-13 08-37-17]
(https://user-images.githubusercontent.com/52042236/218472695-1c64d215-74b7-4828-a0b7-1ced3e3f85b4.png)

**Comment by BravoCharlie-ai** [comment]
Thank you, My issue is resolved actually.

**Comment by huangmang** [comment]
> Thank you, My issue is resolved actually.

hello I met the same problem .I don't see 3d detection result.Cou you tell me how to solve this problem,thanks a lot.

**Comment by huangmang** [comment]
> 
hello, my test result dosen't have 3d ,have you met this problem ？

**Comment by huangmang** [comment]
> Yes. I did inference the KITTI test set with official PV-RCNN weight, and submit the results to KITTI website. The results show as: Car (3D Detection) | 66.01 % | 64.26 % | 61.65 % Pedestrian (3D Detection) | 35.58 % | 29.18 % | 27.36 % Cyclist (3D Detection) | 51.01 % | 42.79 % | 39.31 % Compared with mmlab-PVRCNN model on KITTI leaderboard, it drops almost 10~20 mAP. Any answer to this question?

hello, my test result dosen't have 3d ,have you met this problem ？
![image](https://user-images.githubusercontent.com/50292542/223061095-369766bf-1755-47ee-8067-e687014abc49.png)

**Comment by NiranjanRavi1993** [comment]
Hi all, I am facing the same issue. Pre-trained model/official reported results from pcdet repos are dropping by 1-3 mAP when submission to Kitti test set evaluation. Anyone was able to find a reason/solution?

**Comment by Petros626** [comment]
> aakashjuseja
help this user with your solution @huangmang

**Comment by Petros626** [comment]
@aakashjuseja-aj help him @huangmang
